object-storage-aws/src/test/groovy/io/micronaut/objectstorage/aws/AbstractAwsS3Spec.groovy

```diff
@@ -9,11 +9,12 @@ import software.amazon.awssdk.services.s3.S3Client
 import software.amazon.awssdk.services.s3.model.CreateBucketRequest
 import software.amazon.awssdk.services.s3.model.DeleteBucketRequest
 
+import static io.micronaut.objectstorage.aws.AwsS3Configuration.PREFIX
+
 abstract class AbstractAwsS3Spec extends ObjectStorageOperationsSpecification implements TestPropertyProvider {
 
     public static final String BUCKET_NAME = System.currentTimeMillis()
-
-    public static final String OBJECT_STORAGE_NAME = "default"
+    public static final String OBJECT_STORAGE_NAME = 'default'
```

Comment:
For my own benefit: what's the rationale behind this change? If I'm not mistaken, in both cases the type will be `java.lang.String`. Is it the single quoted variant faster?

Reply:
My preference is to use single quotes for Strings to make it clear that it's not a GString. No performance difference